aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/operation/AWSS3StorageDownloadFileOperation.java

```diff
@@ -85,6 +85,7 @@ public void start() {
                         exception,
                         "See included exception for more details and suggestions to fix."
                 ));
+                return;
```

Comment:
this was resolved in my storage PR, which isn't merged yet. I guess you can keep it in here, but it'll cause a merge conflict later :/